editor/editor_log.h

```diff
@@ -145,11 +145,10 @@ class EditorLog : public HBoxContainer {
 	Button *show_search_button = nullptr;
 	LineEdit *search_box = nullptr;
 
-	// Reference to the "Output" button on the toolbar so we can update it's icon when
-	// Warnings or Errors are encounetered.
+	// Reference to the "Output" button on the toolbar so we can update its icon when warnings or errors are encountered.
 	Button *tool_button = nullptr;
 
-	bool is_loading_state = false; // Used to disable saving requests while loading (some signals from buttons will try trigger a save, which happens during loading).
+	bool is_loading_state = false; // Used to disable saving requests while loading (some signals from buttons will trigger a save, which happens during loading).
```

Comment:
```suggestion
	bool is_loading_state = false; // Used to disable saving requests while loading (some signals from buttons will try to trigger a save, which happens during loading).
```
Alternative fix, probably what was originally meant